settings.sample.yaml

```diff
@@ -0,0 +1,12 @@
+SUBSCRIPTION:
+  RHN_USERNAME: <RHN_USERNAME>
+  RHN_PASSWORD: <RHN_PASSWORD>
+  FM_RHN_POOLID: <FM_RHN_POOLID>
+  DOGFOOD_URL: <DOGFOOD_URL>
+  DOGFOOD_ORG: <DOGFOOD_ORG>
+  DOGFOOD_ACTIVATIONKEY: <DOGFOOD_ACTIVATIONKEY>
+  CAPSULE_DOGFOOD_ACTIVATIONKEY: <CAPSULE_DOGFOOD_ACTIVATIONKEY>
+URLS:
+  HOTFIX_URL: <HOTFIX_URL>
+SERVER:
+  SERVER_HOSTNAME: <SERVER_HOSTNAME>
```

Comment:
Keep all of them commented, so that the value wont pass as a default value.

```suggestion
# SUBSCRIPTION:
  # RHN_USERNAME: <RHN_USERNAME>
  # RHN_PASSWORD: <RHN_PASSWORD>
  # FM_RHN_POOLID: <FM_RHN_POOLID>
  # DOGFOOD_URL: <DOGFOOD_URL>
  # DOGFOOD_ORG: <DOGFOOD_ORG>
  # DOGFOOD_ACTIVATIONKEY: <DOGFOOD_ACTIVATIONKEY>
  # CAPSULE_DOGFOOD_ACTIVATIONKEY: <CAPSULE_DOGFOOD_ACTIVATIONKEY>
# URLS:
  # HOTFIX_URL: <HOTFIX_URL>
# SERVER:
  # SERVER_HOSTNAME: <SERVER_HOSTNAME>
```